packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -26,6 +26,8 @@ export class ProgressBar {
   @Prop() busy?: boolean = false;
   /** (required) Progress bar percentage */
   @Prop() percentage: number = 0;
+  /** (optional) Progress bar percentage to start the animation from (default: 0) */
+  @Prop() percentage_start: number = 0;
```

Comment:
We use camelCase for props, if you could change this, it should be good to merge 🤓

Reply:
Sure, done.

Reply:
Thank you!